variables.tf

```diff
@@ -33,6 +33,24 @@ variable "ami_clean_tag_value" {
   type        = string
 }
 
+variable "ami_clean_unused" {
+  default     = ""
+  description = "Only purge AMIs for which no running instances were built from."
+  type        = string
+}
+
+variable "ami_clean_region" {
```

Comment:
The ami cleaner can be run as a command line tool as well as in a Lambda. The region and profile flags are really only needed for the CLI usage and no for the Lambda

Reply:
Removed profile and region. Thanks!